src/spec.c

```diff
@@ -810,6 +864,19 @@ static int parseVectorField(IndexSpec *sp, StrongRef sp_ref, FieldSpec *fs, Args
   }
 }
 
+static int parseGeometryField(IndexSpec *sp, FieldSpec *fs, ArgsCursor *ac, QueryError *status) {
```

Comment:
This function always return 1, is it correct?

Reply:
Yes
I agree that we don't need it.